llvm/test/Transforms/SROA/propagate-fast-math-flags-on-phi.ll

```diff
@@ -0,0 +1,57 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 4
+; RUN: opt < %s -passes='sroa' -S | FileCheck %s
+define double @phi_with_nsz(double %x) "no-signed-zeros-fp-math" {
```

Comment:
the correct form is "no-signed-zeros-fp-math"="true". The verifier should really check this (or we should start auto upgrading to drop the pointless bool value)